MLIR.cmake

```diff
@@ -139,7 +142,14 @@ function(add_onnx_mlir_library name)
     )
 
   if (ARG_LINK_LIBS)
```

Comment:
You might need ARG_LINK_TARGET_ARCH_LIBS but not ARG_LINK_LIBS. Please separate them.